src/helpers/assertStringsWithinText.ts

```diff
@@ -0,0 +1,34 @@
+import { ReactTestRendererNode } from 'react-test-renderer';
+
+export const assertStringsWithinText = (
```

Comment:
nit: lets match naming with our option name and `renderWithStringValidation`.
```
function validateStringsRenderedInText()
// or 
function validateStringsWithinText()
// or 
function validateStringsRenderedWithinText()
```

Reply:
Good idea ! I renamed it validateStringsRenderedWithinText